nipype/interfaces/ants/utils.py

```diff
@@ -225,3 +225,46 @@ class AffineInitializer(ANTSCommand):
 
     def _list_outputs(self):
         return {'out_file': os.path.abspath(self.inputs.out_file)}
+
+
+class ComposeMultiTransformInputSpec(ANTSCommandInputSpec):
+    dimension = traits.Enum(3, 2, argstr='%d', usedefault=True, mandatory=True,
+                            position=0, desc='image dimension (2 or 3)')
+    output_transform = File(argstr='%s', mandatory=True, position=1,
+                                   desc='Outputfname.txt: the name of the resulting transform.')
+    reference_image = File(argstr='%s', mandatory=False, position=2,
+                                   desc='Reference image (only necessary when output is warpfield)')
+    transforms = InputMultiPath(File(exists=True), argstr='%s', mandatory=True,
+                                position=3, desc='transforms to average')
+
+
+class ComposeMultiTransformOutputSpec(TraitedSpec):
+    output_transform = File(exists=True, desc='Composed transform file')
+
+
+class ComposeMultiTransform(ANTSCommand):
+    """
+    Examples
+    --------
+    >>> from nipype.interfaces.ants import ComposeMultiTransform
+    >>> compose_transform = ComposeMultiTransform()
+    >>> compose_transform.inputs.dimension = 3
+    >>> compose_transform.inputs.transforms = ['struct_to_template.mat', 'func_to_struct.mat']
+    >>> compose_transform.inputs.output_transform = 'func_to_template.mat'
+    >>> compose_transform.cmdline # doctest: +ALLOW_UNICODE
+    'ComposeMultiTransform 3 func_to_template.mat struct_to_template.mat func_to_struct.mat'
+    """
+    _cmd = 'ComposeMultiTransform'
+    input_spec = ComposeMultiTransformInputSpec
+    output_spec = ComposeMultiTransformOutputSpec
+
+    def _format_arg(self, opt, spec, val):
```

Comment:
This should not be necessary, please remove.

Reply:
If I don't include this I get an error:

```traits.trait_errors.TraitError: Cannot set the undefined 'dimension' attribute of a 'ANTSCommandInputSpec' object.```
```/src/nipype/nipype/interfaces/ants/utils.py:256: UnexpectedException```

Maybe this functionality is not yet implemented for ```ANTSCommand```?

Reply:
I think that'd be fixed by removing `mandatory=True` from the `dimension` input trait.

Reply:
No :(.

Reply:
Yay! (it works on my settings at least :+1: )

Reply:
Cool! This works for me too. Maybe it was too late yesterday :).